test/scales/tickGeneratorsTests.ts

```diff
@@ -22,35 +23,62 @@ describe("Scales", () => {
       });
 
       it("generates ticks for a domain crossing 0", () => {
-        let start = -1.5, end = 1, interval = 0.5;
+        let start = -1.5;
+        let end = 1;
         scale.domain([start, end]);
-        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(interval)(scale);
+        let ticks = Plottable.Scales.TickGenerators.intervalTickGenerator(0.5)(scale);
```

Comment:
Do you mean using one interval value for the entire describe block?  
this aligns with our conversation regarding "randomness" of the environment setting, so that we kinda test for integer interval, interval less than one, interval greater than one, etc...